core-bundle/contao/classes/DataContainer.php

```diff
@@ -1197,14 +1201,14 @@ protected function generateHeaderButtons($arrRow, $strPtable)
 					$params['popup'] = '1';
 				}
 
-				$href = System::getContainer()->get('router')->generate($v['route'], $params);
+				$href = System::getContainer()->get('router')->generate($config['route'], $params);
 			}
-			else
+			elseif (isset($config['href']))
 			{
-				$href = $this->addToUrl($v['href'] . '&amp;id=' . $arrRow['id'] . (Input::get('nb') ? '&amp;nc=1' : '') . ($isPopup ? '&amp;popup=1' : ''));
+				$href = $this->addToUrl(($config['href'] ?? '') . '&amp;id=' . $arrRow['id'] . (Input::get('nb') ? '&amp;nc=1' : '') . ($isPopup ? '&amp;popup=1' : ''));
```

Comment:
Why `($config['href'] ?? '')`? The value is always set due to the `elseif (isset($config['href']))` above.